lib/requester/core-body-builder.js

```diff
@@ -187,21 +188,35 @@ module.exports = {
      * @returns {Object}
      */
     raw (content, request) {
-        var contentLanguage = _.get(request, 'body.options.raw.language', 'text');
+        var contentLanguage = _.get(request, 'body.options.raw.language', 'text'),
+            contentTypeHeader;
+
+        if (request && oneNormalizedHeader(request.headers, CONTENT_TYPE_HEADER_KEY)) {
+            contentTypeHeader = oneNormalizedHeader(request.headers, CONTENT_TYPE_HEADER_KEY);
+        }
 
         // Add `Content-Type` header from body options if not set already
-        if (request && !oneNormalizedHeader(request.headers, CONTENT_TYPE_HEADER_KEY)) {
+        if (request && !contentTypeHeader) {
             request.headers.add({
                 key: CONTENT_TYPE_HEADER_KEY,
                 value: CONTENT_TYPE_LANGUAGE[contentLanguage] || CONTENT_TYPE_LANGUAGE.text,
                 system: true
             });
+
+            contentTypeHeader = oneNormalizedHeader(request.headers, CONTENT_TYPE_HEADER_KEY);
```

Comment:
```
if (request && oneNormalizedHeader(request.headers, CONTENT_TYPE_HEADER_KEY)) {
            contentTypeHeader = oneNormalizedHeader(request.headers, CONTENT_TYPE_HEADER_KEY);
}
```

Should above code need to be below this line 206 so that there wont be a need reassign it again in line  206